### PR TITLE
Attempt

### PR DESCRIPTION
**Pull request by brendenbarta**

### CODE REVIEW

File: js-intro.js [hunk 1]
```diff
@@ -1,43 +1,84 @@
...
 
 
 
+// 1d. STRETCH: Write a function that can take two arguments - a string and a letter and returns a sentence telling the user if the letter exists within the string.
+const existanceReporter = (text, testletter) =>{
+  if(testLetter.tolowercase().includes(text)){
+    return `congrats!` ${testLetter} exists in the string
+  }
+}
 // ------------------------------ Consider the following variables:
```
Comment (sjproctor) [inline js-intro.js:44]: Nice job!

File: js-intro.js [hunk 2]
```diff
@@ -1,43 +1,84 @@
...
+  } else {
+    return  myCat
+  }
+}
 
+}
 
+console.log(sizer(myDog,myCat))
 
-// 2b. Create a function that takes in two arguments and returns an array containing the two variables. Expected output: ["Snoopy", "Garfield"]
```
Comment (brendenbarta) [inline js-intro.js:65]: oops I forgot to add this part X(

File: js-intro.js [hunk 3]
```diff
@@ -84,13 +136,36 @@ var testString = "thisisateststring"
...
 // 5a. Write a function that takes in the toonimals array as an argument and returns an array with only the toon objects that are cats. Expected output: [ { name: "Stimpy", animal: "cat" }, { name: "Scratchy", animal: "cat" }, { name: "Felix", animal: "cat" } ]
-
-
+const catFinder = (objects) =>{
+ const nonCats = objects.filter((object) => {
+  return object.animal ! == 'cat'
+ })
+ return nonCats.mao((object) => object.name)
+}
+ console.log("cat finder, catFinder(toonimals"))
```
Comment (sjproctor) [inline js-intro.js:147]: Really close, just a few tweaks to make this find all the cats:
const catFinder = (objects) =>{
 return objects.filter((object) => {
  return object.animal === 'cat'
 })
}
 console.log("cat finder", catFinder(toonimals))

File: js-intro.md [hunk 4]
```diff
@@ -5,49 +5,51 @@ Answer the following questions. First, without external resources. Challenge you
...
 
 
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-  Your answer:
+  Your answer: 2 people working on one project, giving equal turns. there is always a driver and a navigator, the navigator gives advice and assitance while the driver types.
 
-  Researched answer:
+  Researched answer:pair programming includes two individuals (developers) working on a single system, with one typing and the other one navigating or observing. Generally the one responsible for typing is known as the ‘Driver’, while the other one called the ‘Navigator’ continually revise and review what is being coded or typed. During the whole of their time with each other, the duo invariably keeps in touch, enabling the other partner to participate and help outline the code direction.
```
Comment (sjproctor) [inline js-intro.md:45]: Nice!

**Review by sjproctor — APPROVED** [review]
Great job! Keep studying and reviewing, you are doing really well.

**Pull request merged**